src/pages/user/review/[username].tsx

```diff
@@ -1,57 +1,223 @@
+              className="h-6 w-6 fill-yellow stroke-yellow"
+              strokeWidth={2}
+            ></StarIcon>
+          );
+        })}
+        {[...Array<Number>(5 -Math.round(rating.rating)),].map((e, idx) => {
+          return (
+            <StarIcon
+              key={idx}
+              className="h-6 w-6 fill-none stroke-yellow"
+              strokeWidth={2}
+            ></StarIcon>
+          );
+          })}
+      </div>
+    )
+  }
 
-  return (
-    <main className="w-full h-screen pt-20 pl-6 flex flex-col bg-backgroundColor">
-      <div className="flex items-center mb-3 md:mb-8">
-        <BackButton href="/user/booking" />
-        <p className="text-xl md:text-3xl mx-5 md:mx-10 font-bold display: inline-flex">
-          Rating and Review
-        </p>
+  const EmptyStar = () => {
+    return (
+      <div>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="2"
+          stroke="currentColor"
+          aria-hidden="true"
+          className="h-10 w-10 none-yellow stroke-slate-300"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
+          ></path>
+        </svg>
       </div>
-      <div className="flex h-full">
-        <div className="w-2/5 flex-col mr-10">
-          <div className="flex flex-col items-center justify-start px-2 md:px-5 h-full text-2xl">
-            Trainer Name
-            <div className="relative w-52 h-52 md:w-60 md:h-60 lg:w-80 lg:h-80 mt-6 md:mt-10 rounded-2xl overflow-hidden">
-              <Image
-                src="/default_profile.jpg"
-                alt=""
-                fill
-                sizes="(max-width: 768px) 100vw"
-                style={{ objectFit: "contain"}}
-              />
-            </div>
+    );
+  };
+
+  const ReviewBox = (review:ReviewDetail) => {
+    return (
+      <div className="h-[20%] w-[80%] bg-white mb-[5%] border-2 border-gray rounded-3xl items-center justify-between drop-shadow-lg hover:bg-gray-light">
+        <div className="m-5">
+          <div className="text-lg">
+            <StarRating rating={review.rating}></StarRating>
+          </div>
+          <div className="text-lg">
+            comment : {review.comment}
+          </div>
+        </div>  
+      </div>
+    );
+  };
+
+  
+
+  return (
+    <>
+      <div className="w-full h-screen flex flex-row items-center justify-center text-3xl bg-backgroundColor">
+        <div className="w-2/5 h-screen flex-col">
+          <div className="mt-[20%]"></div>
```

Comment:
I already fix the padding and position of main items in this page, you should look at main branch because this will overlap with navbar.